.circleci/config.yml

```diff
@@ -0,0 +1,128 @@
+defaults:
+  - &default-config
+    parallelism: 1
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      BUNDLE_PATH: vendor/bundle
+    macos:
+      xcode: "9.2.0"
+    shell: /bin/bash --login -eo pipefail
+  - &prepare-storage
+    run:
+      name: Create directories for artifacts and reports
+      command: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+  - &restore-gems
+    restore_cache:
+      keys:
+        - v1-gems-{{ checksum "Gemfile.lock" }}
+        - v1-gems-
+  - &install-gems
+    run:
+      name: Bundle install
+      command: bundle check || bundle install
+      environment:
+        BUNDLE_JOBS: 4
+        BUNDLE_RETRY: 3
+  - &store-gems
+    save_cache:
+      key: v1-gems-{{ checksum "Gemfile.lock" }}
+      paths:
+        - vendor/bundle
+  - &fetch-xcode-logs
+    run:
+      name: Getting Xcode activity logs
+      command: find $HOME/Library/Developer/Xcode/DerivedData -name '*.xcactivitylog' -exec cp {} $CIRCLE_ARTIFACTS/xcactivitylog \; || true
```

Comment:
This command doesn't seem to be referenced anywhere else in the config file?

Reply:
Huh, missed that. For now it's only needed in the `build-and-test` job, but could be used in more jobs if needed.